test/integration/com/unifina/service/PermissionServiceIntegrationSpec.groovy

```diff
@@ -328,16 +326,6 @@ class PermissionServiceIntegrationSpec extends IntegrationSpec {
 		e.message == "Cannot revoke only SHARE permission of ${stream}"
 	}
 
-	void "getPermissionsTo(resource, userish) returns correct UI channel read permissions via associated dashboard"() {
-		service.systemGrantAll(me, dashboard)
-		def permissions = service.getPermissionsTo(uiChannelStream, me)
-
-		expect:
-		permissions.size() == 1
-		permissions.get(0).operation == Permission.Operation.READ
```

Comment:
(Related to previous comments in this file:)

Here you can see the old test, which grants access to dashboard and then checks whether the user has `READ` permission to the `uiChannelStream`. In the new permission system `READ` becomes `STREAM_GET` and `STREAM_SUBSCRIBE`.